src/components/jellyfinActions.ts

```diff
@@ -199,12 +198,22 @@ export function pingTranscoder(
  */
 export function load(
     $scope: GlobalScope,
-    customData: PlaybackProgressInfo,
+    customData: any,
```

Comment:
sorry in advance..
as mentioned earlier, extend was able to put properties on customData that did not even belong in the PlaybackProgressInfo type, so in reality customData is a bit of an extended type. I think we should get rid of $scope and clean up the types from there.